minecraft_servers/superawesome/manifest.json

```diff
@@ -7,7 +7,18 @@
       "%.superawesomeminecraftserver.com"
     ],
     "social": {
+      "web_shop": "http://superawesome.buycraft.net/",
+      "discord": "https://discord.gg/SuperAwesome",
+      "tiktok": "superawesome.dk",
       "youtube": "https://www.youtube.com/channel/UClhOqD9GXk2zGHZRluDLIBQ",
-      "discord": "https://discord.gg/QAem6Yq"
+    },
+    "brand": {
+      "primary": "#306fdb",
+      "background": "#306fdb",
+      "text": "#ffffff"
+    },
+    "discord": {
+      "server_id": 313097609948430336,
+      "rename_to_minecraft_name": true
```

Comment:
Only staff members should create pull requests with this value = `true`. Please remove the line.